stubs/protobuf/google/protobuf/descriptor.pyi

```diff
@@ -60,6 +60,27 @@ class Descriptor(_NestedDescriptorBase):
     oneofs: Any
     oneofs_by_name: Any
     syntax: Any
+    def __init__(
+        self,
+        name: str,
+        full_name: str,
+        filename: Any,
+        containing_type: Descriptor,
+        fields: list[FieldDescriptor],
+        nested_types: list[FieldDescriptor],
+        enum_types: list[EnumDescriptor],
+        extensions: list[FieldDescriptor],
+        options: Any | None = ...,
+        serialized_options: Any | None = ...,
+        is_extendable: bool = ...,
+        extension_ranges: Any = ...,
+        oneofs: list[OneofDescriptor] = ...,
+        file: FileDescriptor = ...,
+        serialized_start: Any = ...,
+        serialized_end: Any = ...,
+        syntax: str = ...,
+        create_key: Any = ...,
+    ): ...
```

Comment:
this makes sense. In the future, would appreciate a link to the original source to help double check. I ended up finding it here

https://github.com/protocolbuffers/protobuf/blob/main/python/google/protobuf/descriptor.py#L319